source/js/components/nav/mobile-dropdown.js

```diff
@@ -63,30 +58,20 @@ class NavMobileDropdown extends Accordion {
   }
 
   open() {
-    super.open();
```

Comment:
Summary of changes in this file:

- Adding sliding effect by animating section's height
- We've decided to allow multiple open accordions at the same item. There's no longer needs to reset sibling sections.